chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json

```diff
@@ -1,98 +1,508 @@
 {
 	"info": {
-		"_postman_id": "21c26d32-3ea4-4d99-ac3d-c4c557baa398",
+		"_postman_id": "ec8c7252-3caa-48d3-8060-d247a94d6ea3",
```

Comment:
Is this line needed? It seems to change often, but I don't know what it's used for. If it's metadata kinds of stuff do we need to track it at all?

Reply:
Tests are running successfully without postman_id. So removed it.